config/boards/h96-tvbox-3566.tvb

```diff
@@ -0,0 +1,12 @@
+# Rockchip RK3566 quad core 8GB RAM SoC WIFI/BT eMMC USB2/3
+BOARD_NAME="h96-tvbox-3566"
+BOARDFAMILY="rk35xx"
+BOARD_MAINTAINER="tdleiyao"
```

Comment:
if you are adding new config, this should be you or empty

Reply:
Changed the mantainer name